infra/nightly-resources/chart.js

```diff
@@ -112,11 +131,6 @@ function initializeChart() {
     type: "bar",
     data: chartData,
     options: {
-      scales: {
-        y: {
-          beginAtZero: true,
```

Comment:
is this bc it wasn't working before?

Reply:
Yeah I copied some of this code from the ChartJS docs, but I don't think this was actually doing anything